DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>Add a bit more explanation here about tangential and cross ellipticities. Maybe we could consider embedding a PNG with a figure or cartoon with these, mayeb Fig. 5 of recent review by Judit Pratt (<a href="https://arxiv.org/pdf/2501.07938" target="_blank">https://arxiv.org/pdf/2501.07938</a>).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>Adjusted</p>